tutorials/aperiodic/plot_Fluctuations.py

```diff
@@ -0,0 +1,153 @@
+
+###################################################################################################
+# Applying DFA
+# ------------
+#
+# Here, to introduce DFA, we will use colored noise signals, (white noise and pink noise),
+# with have different auto-correlation properties, and so should show different DFA results,
+# which we can compare.
+#
+# Note that DFA can be applied to multiple signal types, and in neural field data it is
+# often used to examine amplitude time series of neural oscillations.
+#
+
+###################################################################################################
+
+# Simulation settings settings
+n_seconds = 10
+fs = 500
+
+# Simulate a white noise signal
+sig_wn = sim_powerlaw(n_seconds, fs, exponent=0)
+
+# Simulate a pink noise powerlaw signal
+sig_pl = sim_powerlaw(n_seconds, fs, exponent=-1)
+
+###################################################################################################
+# Calculating DFA
+# ~~~~~~~~~~~~~~~
+#
+# The DFA algorithm involves:
+#
+# 1. Removing the mean of a signal (detrend)
+# 2. Computing the cumulative sum of the signal
+# 3. Splitting the signal in equal-sized windows
+# 4. Fitting a polynomial across the windows
+# 5. Calculate the mean squared residual (fluctuation) of the fit
+#
+# Steps 3-5 are repeated for various window sizes.
+#
+# The DFA algorithm is available through the :func:`~.compute_fluctuations` function.
+#
+# Algorithm Settings
+# ~~~~~~~~~~~~~~~~~~
+#
+# The DFA algorithm requires certain settings, including:
+#
+# - `n_scales` : the number of scales to estimate fluctuations over.
+# - `min_scale` : the shortest scale, in seconds, to compute over.
+# - `max_scale` : the longest scale, in seconds, to compute over.
+#
+
+###################################################################################################
+
+# Compute DFA for a white noise signal
+ts_wn, flucs_wn, exp_wn = compute_fluctuations(sig_wn, fs, n_scales=10,
+                                               min_scale=0.01, max_scale=1.0)
+
+# Compute DFA for a pink noise signal
+ts_pl, flucs_pl, exp_pl = compute_fluctuations(sig_pl, fs, n_scales=10,
+                                               min_scale=0.01, max_scale=1.0)
+
+###################################################################################################
+# Results
+# ~~~~~~~
+#
+# The :func:`~.compute_fluctuations` function returns the time scales and
+# measured fluctuations from the DFA analysis.
+#
+# To see what these look like, in the plot below, fluctuations are plotted across window
+# sizes for both signals in log-log space.
+#
+
+###################################################################################################
+
+_, ax = plt.subplots(figsize=(5, 5))
+
+ax.loglog(ts_wn, flucs_wn, label="White Noise")
+ax.loglog(ts_pl, flucs_pl, label="Power Law")
+
+ax.set(title="Fluctuations Across Window Sizes",
+       xlabel="Time Scales", ylabel="Fluctuations")
+plt.legend();
```

Comment:
Why is a semi-colon here?

Reply:
these mpl utility functions return some nonsense, which gets printed out in the sphinx site and/or downloaded notebooks. This is just to supress output.